### PR TITLE
[fixes #1950] Cliqz product page

### LINKED ISSUE #1950: Create Cliqz product page
This is blocked by receipt of final copy and assets.

Ref #1941.

### PR DESCRIPTION
**Pull request by ckprice**
Preview: https://pageshot.net/ALI6qEjOUkVUjE07/testpilot.dev

**Blocked** by
- [x] Final sign off on privacy policy document, including metrics.md URL (@javaun, @ellee)
- [x] Provide a new file for tour step 3 that has just "Cliqz" as the name (@johngruen)
- [x] Confirm contributor information looks okay (@javaun)
- [ ] Confirm gradient colors (@wbowden)
- [ ] Getting [an error](https://cloud.githubusercontent.com/assets/8632184/21572181/33ac577c-ce8b-11e6-94ec-1ee0aca75b6a.png) when trying to enable (@clouserw)
- [ ] Confirm incompat [add-on description](https://cloud.githubusercontent.com/assets/8632184/21572207/7448eeee-ce8b-11e6-9a19-09b90ac4b7eb.png) for users using the main Cliqz add-on. Ref #1941 (@javaun, @johngruen)

Notes
- Using the `locale_grantlist` is correctly hiding the card on the homepage from non-de, but non-de users are still able to deeplink to the page. Not sure if this is a problem.

/cc @flodolo 

Fixes #1950, #1941.

### CODE REVIEW

**Comment by ghost** [comment]
I double checked the hashes on https://testpilot.firefox.com/files/testpilot@cliqz.com/latest and they are right and the file downloads for me.  I noticed the file has a max version of 51.* so it won't run in nightly.  I don't think that would cause that error though.  Anything in the console?

**Comment by johngruen** [comment]
@clouserw seems like that max-version might be an issue since ~~i think~~ 51 is beta RN. We should confirm if this can be changed.

**Comment by ckprice** [comment]
> Using the `locale_grantlist` is correctly hiding the card on the homepage from non-de, but non-de users are still able to deeplink to the page. Not sure if this is a problem.

Confirmed in checkin today that this won't be a problem. I thought about using the `locales` to at least display a warning for non-de users, but it isn't working correctly afaict (ref #1959).

**Comment by ckprice** [comment]
I think we can go ahead and review/merge this to get localization moving.

**Review by johngruen — APPROVED** [review]

**Comment by johngruen** [comment]
@ckprice lgtm, merge at your leisure

**Comment by flodolo** [comment]
What's going to move strings into the .ftl file? Because that's the only way for locales to see these strings.

**Comment by chuckharmston** [comment]
> What's going to move strings into the .ftl file? Because that's the only way for locales to see these strings.

@flod The FTL is generated in the build process from these files:

https://testpilot.firefox.com/static/locales/en-US/experiments.ftl

https://github.com/mozilla/testpilot/blob/master/docs/content/reference.md#localization

**Comment by ckprice** [comment]
PR updated @flodolo @chuckharmston

File: docs/content/reference.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ To make a new experiment:
 
 An experiment's YAML file is considered to contain the canonical localization for English, and is compiled into [an FTL file](https://testpilot.dev.mozaws.net/static/locales/en-US/experiments.ftl) with each build. That file is monitored by Pontoon, and strings are translated by volunteer localizers.
 
+For new experiments, you must copy the build version of `expeirments.ftl` into the `locales` directory, and include it in the pull request.
```
Comment (flodolo) [inline docs/content/reference.md]: Typo: experiments.ftl

**Comment by ckprice** [comment]
Going to go ahead and self-merge in a bit if there are no other comments. It's timely.

Jgruen r+'d before the FTL addition, and it's working for me locally.

**Pull request merged**